From: "Moger, Babu" <Babu.Moger@netapp.com>
Date: Tue, 4 Sep 2012 17:04:33 +0000
Subject: scsi_dh_rdac: Add a new netapp vendor/product string
References: bnc#772483
Git-commit: e7ba29d6332850512ad45f154cfd5601fea39f85
Patch-Mainline: v3.7

This patch adds a new vendor/product strings for netapp E series  product.
Also consolidated the strings together with similar names.

Signed-off-by: Babu Moger <babu.moger@netapp.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Acked-by: Hannes Reinecke <hare@suse.de>

diff --git a/drivers/scsi/device_handler/scsi_dh_rdac.c b/drivers/scsi/device_handler/scsi_dh_rdac.c
index 20c4557..1fc6438 100644
--- a/drivers/scsi/device_handler/scsi_dh_rdac.c
+++ b/drivers/scsi/device_handler/scsi_dh_rdac.c
@@ -795,24 +795,25 @@ static const struct scsi_dh_devlist rdac_dev_list[] = {
 	{"SGI", "TP9700"},
 	{"SGI", "IS"},
 	{"STK", "OPENstorage D280"},
+	{"STK", "FLEXLINE 380"},
+	{"SUN", "CSM100_R_FC"},
 	{"SUN", "CSM200_R"},
 	{"SUN", "LCSM100_I"},
 	{"SUN", "LCSM100_S"},
 	{"SUN", "LCSM100_E"},
 	{"SUN", "LCSM100_F"},
+	{"SUN", "STK6580_6780"},
+	{"SUN", "SUN_6180"},
+	{"SUN", "ArrayStorage"},
 	{"DELL", "MD3000"},
 	{"DELL", "MD3000i"},
 	{"DELL", "MD32xx"},
 	{"DELL", "MD32xxi"},
 	{"DELL", "MD36xxi"},
 	{"DELL", "MD36xxf"},
+	{"NETAPP", "INF-01-00"},
 	{"LSI", "INF-01-00"},
 	{"ENGENIO", "INF-01-00"},
-	{"STK", "FLEXLINE 380"},
-	{"SUN", "CSM100_R_FC"},
-	{"SUN", "STK6580_6780"},
-	{"SUN", "SUN_6180"},
-	{"SUN", "ArrayStorage"},
 	{NULL, NULL},
 };
 
